nebu/cli/get.py

```diff
@@ -29,6 +27,10 @@
 def get(ctx, env, col_id, col_version, output_dir, book_tree, get_resources):
     """download and expand the completezip to the current working directory"""
 
+    # Set book-tree default from config
+    if book_tree is None:
+        book_tree = ctx.obj['settings'].get('default_format') == 'tree'
+
```

Comment:
I realize this is unrelated to the linked issue - I had a branch that put the 'pick a default for neb get format' already lying around, so started with it, since I needed the extended config parsing. Think of it as a freebie.